src/Avalonia.Controls/Primitives/ToggleButton.cs

```diff
@@ -49,12 +49,9 @@ public class ToggleButton : Button
 
         private bool? _isChecked = false;
 
-        static ToggleButton()
+        public ToggleButton()
         {
-            PseudoClass<ToggleButton, bool?>(IsCheckedProperty, c => c == true, ":checked");
-            PseudoClass<ToggleButton, bool?>(IsCheckedProperty, c => c == false, ":unchecked");
-            PseudoClass<ToggleButton, bool?>(IsCheckedProperty, c => c == null, ":indeterminate");
-
+            UpdatePseudoClasses(IsChecked);
             IsCheckedProperty.Changed.AddClassHandler<ToggleButton>((x, e) => x.OnIsCheckedChanged(e));
```

Comment:
Should be in static ctor since this will cause each created `ToggleButton` derived control to add one more `IsChecked` class handler.

Reply:
Good catch: was the result of a bad manual merge. Fixed.